### DISCUSSION TITLE
Please update API docs - Some references are missing

### DISCUSSION THREAD

**Discussion by esturniolo** [semaphoreui/semaphore#1839] [General]
Hi all.
I'm tried to follow the exact steps of the examples of the ["Updates inventory"](https://www.semui.co/api-docs/#/project/put_project__project_id__inventory__inventory_id_) but when you execute it, it says `{"error":"Inventory ID in body and URL must be the same"}
`

So I must add the `id` in the body to run this. When I added it, worked like a charm.
The thing is that this isn't mentioned ion the docs and may be this can be happening in others requests of the API docs.

**Comment by fiftin** [comment]
Hi @esturniolo you are right, will fix soon. Thanks a lot!

**Reply by fiftin** [reply]
All PUT requests require `id` in the body.

I will think, perhaps we will change this behaviour.

**Reply by fiftin** [reply]
Thank you @esturniolo You helped a lot.
